Exercise_D and E/index.html

```diff
@@ -10,7 +10,55 @@
   </head>
   <body>
     <!-- CODE HERE -->
-
+    <main class="gridcontainer">
+      <div class="topbar">
+        <img
+          src="/Exercise_D and E/assets/images/logo.png"
+          alt="Flowershop logo"
+        />
+      </div>
+      <div class="midbackdrop"></div>
+      <div class="mid">
+        <h1 class="header">Beautiful Flowers</h1>
+        <button class="button">Learn more</button>
+      </div>
+      <div class="bottombox1">
+        <div class="flowerwrapper">
+          <img
+            id="flowericon"
+            src="../Exercise_D and E/assets/images/flower-icon.png"
+            alt="Flower Icon"
+          />
+        </div>
+        <div class="flowerphrase">Flowers for every<br />kind of situation</div>
+        <div class="moneywrapper">
+          <img
+            id="moneyicon"
+            src="/Exercise_D and E/assets/images/payment-icon.png"
+            alt="Money Icon"
+          />
+        </div>
+        <div class="moneyphrase">All major payments<br />kinds accepted</div>
+      </div>
+      <div class="bottombox2">
+        <div class="phonewrapper">
+          <img
+            id="phoneicon"
+            src="/Exercise_D and E/assets/images/phone-icon.png"
+            alt="Phone Icon"
+          />
+        </div>
+        <div class="phonephrase">Free consultation</div>
+        <div class="truckwrapper">
+          <img
+            id="truckicon"
+            src="/Exercise_D and E/assets/images/truck-icon.png"
+            alt="Truck Icon"
+          />
+        </div>
+        <div class="truckphrase">City wide delivery</div>
+      </div>
+    </main>
     <script src="" async defer></script>
   </body>
 </html>
```

Comment:
You have recreated the website exactly like how it's supposed to be with fantastic use of grid, flex, and media query. Excellent work!